SJMaster/urls.py

```diff
@@ -17,34 +17,28 @@
 from django.urls import path, include
 from jobboard.views import board
 from student.views import update_student_account_settings_view, account_update_success
-from recruiter.views import (UpdateRecruiterSettings,
-                             CreateNewJobForm,
-                             job_created_successfully,
-                             recruiterRegister,
-                             recruiter_created_successfully,
-                             recruiter_view_my_jobs_and_applications,)
+from recruiter.views import (
+    CreateNewJobForm,
+    job_created_successfully,
+    recruiterRegister,
+    recruiter_created_successfully,
+    recruiter_view_my_jobs_and_applications,
+    update_recruiter_account_settings_view)
 from student.views import studentRegister, student_created_successfully
 
-
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', board, name='board'),
     path('', include("django.contrib.auth.urls"), name="login"),
-    path('recruiter/account_settings/<slug:pk>',
-         UpdateRecruiterSettings.as_view(), name='recruiter_account_settings'),
     path('recruiter/create_new_job_form/',
```

Comment:
Is this line duplicated on line 52 below? Here you can see the danger of moving code around too much in a PR that is meant to add functionality - I could've missed that...

Also it seems a little strange now that we're importing a form object for a "views" module, not to mention using an object here at all as opposed to a plain function.

Reply:
@ifireball Your right, there were actually a few duplicated lines in the urls file I just fixed them. I will make sure to fix conflicts from now and on more carefully in order to avoid these kind of mistakes.